sdk/keyvault/azure-security-keyvault-secrets/pom.xml

```diff
@@ -43,12 +43,6 @@
       <version>1.1.0</version> <!-- {x-version-update;com.azure:azure-core-http-netty;dependency} -->
     </dependency>
 
-    <dependency>
-      <groupId>commons-codec</groupId>
-      <artifactId>commons-codec</artifactId>
-      <version>1.13</version> <!-- {x-version-update;commons-codec:commons-codec;external_dependency} -->
```

Comment:
Also, update the `version_client.txt` and remove this dependency. I believe this was the only place this dependency was used.

Reply:
I am assuming external_dependencies.txt

Reply:
Track 1 KV still references commons-codec.
So, cannot remove from external_dependencies.txt too,

Reply:
So nothing should ever be deleted from external_dependencies.txt unless you know for an absolute fact it isn't being used by anywhere else. There are several data track libraries that require this dependency. In my comment above I put in a couple of commands that can be run from the command line to check, in the future.